include/tvm/relay/pass.h

```diff
@@ -296,6 +296,26 @@ struct StructuralHash {
   size_t operator()(const Expr& expr) const;
 };
 
+/*! \brief turn a dataflow graph into A Normal Form.
```

Comment:
Let's be a little bit more formal.
```suggestion
/*! \brief turn a dataflow graph into Administrative Normal Form, or A-Normal Form (ANF).
```